client/src/components/App.jsx

```diff
@@ -60,8 +59,7 @@ class App extends React.Component {
     });
 
     this.setState({
-      reviews,
-      rating,
+      reviews, rating,
     });
```

Comment:
this isn't the right area

Reply:
the red parts indicate deleted lines..

Reply:
Line 67-68 of app.jsx. Sorry about the confusion

Reply:
ohh i see. i'll fix it now